File: 349-intersection-of-two-arrays/intersection-of-two-arrays.js
var intersection = function(nums1, nums2) {
    let str = new Set(nums1);
    let string = new Set(nums2);
    let result = []
    
   for(let num of str){
    if(string.has(num)){
        result.push(num)
    }
   }
   return result
};

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.cs
public class Solution {
    public int[] Intersection(int[] nums1, int[] nums2) {
        
        HashSet<int> set1 = new HashSet<int>(nums1);
        HashSet<int> set2   = new HashSet<int>(nums2);

        List<int> result = new List<int>();

        foreach(int num in set1){
            if(set2.Contains(num)){
                result.Add(num);
            }
        }
        return result.ToArray();
    }
}